sdk/websites/Microsoft.Azure.Management.WebSites/src/Generated/Models/StaticSiteARMResource.cs

```diff
@@ -45,23 +45,39 @@ public StaticSiteARMResource()
         /// <param name="repositoryUrl">URL for the repository of the static
         /// site.</param>
         /// <param name="branch">The target branch in the repository.</param>
+        /// <param name="provider">The provider that submitted the last
+        /// deployment to the primary environment of the static site.</param>
         /// <param name="customDomains">The custom domains associated with this
         /// static site.</param>
         /// <param name="repositoryToken">A user's github repository token.
         /// This is used to setup the Github Actions workflow file and API
         /// secrets.</param>
         /// <param name="buildProperties">Build properties to configure on the
         /// repository.</param>
-        public StaticSiteARMResource(string location, string id = default(string), string name = default(string), string kind = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string defaultHostname = default(string), string repositoryUrl = default(string), string branch = default(string), IList<string> customDomains = default(IList<string>), string repositoryToken = default(string), StaticSiteBuildProperties buildProperties = default(StaticSiteBuildProperties), SkuDescription sku = default(SkuDescription))
+        /// <param name="templateProperties">Template options for generating a
+        /// new repository.</param>
+        /// <param name="contentDistributionEndpoint">The content distribution
+        /// endpoint for the static site.</param>
+        /// <param name="keyVaultReferenceIdentity">Identity to use for Key
+        /// Vault Reference authentication.</param>
+        /// <param name="userProvidedFunctionApps">User provided function apps
+        /// registered with the static site</param>
+        public StaticSiteARMResource(string location, string id = default(string), string name = default(string), string kind = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string defaultHostname = default(string), string repositoryUrl = default(string), string branch = default(string), string provider = default(string), IList<string> customDomains = default(IList<string>), string repositoryToken = default(string), StaticSiteBuildProperties buildProperties = default(StaticSiteBuildProperties), StaticSiteTemplateOptions templateProperties = default(StaticSiteTemplateOptions), string contentDistributionEndpoint = default(string), string keyVaultReferenceIdentity = default(string), IList<StaticSiteUserProvidedFunctionApp> userProvidedFunctionApps = default(IList<StaticSiteUserProvidedFunctionApp>), SkuDescription sku = default(SkuDescription), ManagedServiceIdentity identity = default(ManagedServiceIdentity))
```

Comment:
Same here. `templateProperties`, `contentDistributionEndpoint`, `keyVaultReferenceIdentity`, and `userProvidedFunctionApps` should be after `SkuDescription sku = default(SkuDescription)`.

Reply:
@JonathanCrd while trying to make the changes to parameter order that was called out here - the swagger validation is flagging these as breaking changes https://github.com/Azure/azure-rest-api-specs/pull/14018/checks?check_run_id=2422311229 
how do we resolve this?